assets/compat/config_spec.yaml

```diff
@@ -115,21 +115,23 @@ bitcoind:
   name: Bitcoin Core
   description: |
     The Bitcoin Core node to connect to:
+      - none: Use the light bitcoin backend built into LND, Neutrino.
       - internal: The Bitcoin Core and Proxy services installed to your Embassy
-      - external: An unpruned Bitcoin Core node running on a different device
   tag:
     id: type
     name: Type
     variant-names:
+      none: None (Built-in LND Neutrino)
```

Comment:
I think this should be last in the list

Reply:
Interestingly, that's how it already appears. Was actually gonna file a bug about this haha

<img width="791" alt="Screen Shot 2022-06-29 at 5 22 15 PM" src="https://user-images.githubusercontent.com/7445670/176555749-b091211f-542f-41ef-8da0-a95534d7f5e8.png">